lib/helpers.bash

```diff
@@ -96,8 +96,7 @@ bash-it ()
          version)
              func=_bash-it-version;;
          *)
-             reference bash-it
-             return;;
+             func=_bash-it-additional-$verb $component "$@"
```

Comment:
Why make this so complex? Why not simply adding a new case for `reload`, similar to how e.g. `version` is handled?